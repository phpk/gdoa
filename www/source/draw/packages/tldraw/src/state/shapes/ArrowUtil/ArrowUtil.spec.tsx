import { Arrow } from '..'

describe('Arrow shape', () => {
  it('Creates a shape', () => {
    expect(Arrow.create({ id: 'arrow' })).toMatchSnapshot('arrow')
  })
})
