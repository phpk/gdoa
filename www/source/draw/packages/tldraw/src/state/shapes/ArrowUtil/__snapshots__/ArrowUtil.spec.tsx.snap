// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow shape Creates a shape: arrow 1`] = `
Object {
  "bend": 0,
  "childIndex": 1,
  "decorations": Object {
    "end": "arrow",
  },
  "handles": Object {
    "bend": Object {
      "id": "bend",
      "index": 2,
      "point": Array [
        0.5,
        0.5,
      ],
    },
    "end": Object {
      "canBind": true,
      "id": "end",
      "index": 1,
      "point": Array [
        1,
        1,
      ],
    },
    "start": Object {
      "canBind": true,
      "id": "start",
      "index": 0,
      "point": Array [
        0,
        0,
      ],
    },
  },
  "id": "arrow",
  "name": "Arrow",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "rotation": 0,
  "style": Object {
    "color": "black",
    "dash": "draw",
    "isFilled": false,
    "scale": 1,
    "size": "small",
  },
  "type": "arrow",
}
`;
